jupyter/datascience/anaconda-python-3.8/runtime-images/datascience-ubi8-py38.json

```diff
@@ -5,7 +5,7 @@
       "datascience"
     ],
     "display_name": "Datascience with Python 3.8 (UBI8)",
-    "image_name": "quay.io/opendatahub/workbench-images@sha256:471386969e54c287e3d1777479d1e8b05f1e07b824cc2c1ec98d29ac5006b378",
+    "image_name": "quay.io/opendatahub/workbench-images@sha256:a4290926fbfa8c54825ab867b7201c11be8c676a96d4811fdb3a8f1e348d3eb0",
```

Comment:
I don't understand this diff very much. It states that the only change on this line is the hash of the image. Yet current content of that line uses an image from the `modh` repository as expected https://github.com/red-hat-data-services/notebooks/blob/main/jupyter/datascience/anaconda-python-3.8/runtime-images/datascience-ubi8-py38.json#L8 .

We don't want to have images from `opendatahub` image repo in the downstream codebase.